server/webapp/WEB-INF/rails.new/webpack/views/dashboard/pipeline_operations_widget.js.msx

```diff
@@ -89,19 +89,33 @@ const PipelineOperationsWidget = {
       modal.render();
     };
 
+    const getErrorMessageFromStatus = (textStatus) => {
+      if (textStatus === 'parsererror') {
+        return "Error occurred while parsing the data.";
+      }
+      return "There was an unknown error.";
+    };
+
     self.destroyPausePopup = (modal) => {
       self.pauseMessage("");
       modal.destroy();
     };
 
     self.triggerWithOptionsVM   = Stream(new TriggerWithOptionsVM());
     self.triggerWithOptionsInfo = Stream();
+    self.errorMessage           = Stream('');
 
     self.showTriggerWithOptionsPopup = (pipeline) => {
-      TriggerWithOptionsInfo.all(pipeline.name).then((info) => {
-        self.triggerWithOptionsInfo(info);
-        self.triggerWithOptionsVM().initialize(info);
-      }).then(m.redraw);
+      TriggerWithOptionsInfo.all(pipeline.name)
+        .then((info) => {
+          self.triggerWithOptionsInfo(info);
+          self.triggerWithOptionsVM().initialize(info);
+          self.errorMessage('');
+        })
+        .then(m.redraw)
+        .fail((ex, textStatus) => { //eslint-disable-line no-unused-vars
```

Comment:
Do not disable lint. For an unused variable prefix it with an underscore!
`.fail((_res, textStatus)=>{})`